<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>501</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>USER CONTROLS</string>
    </property>
    <widget class="QPushButton" name="Power">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>200</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>POWER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StartSession">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>200</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>START SESSION</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Intensity">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>210</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>190</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Intensity</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Username">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;USERNAME&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Saved sessions</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="LeftEar">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>left earpiece</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="RightEar">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>right earpiece</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>161</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>SESSION</string>
     </property>
     <widget class="QTextEdit" name="RadioCustomText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;0:01:00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Radio45Min">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>45 min</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RadioCustom">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>custom (input below)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Radio20Min">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>20 min</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>111</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>SESSION GROUP</string>
     </property>
     <widget class="QRadioButton" name="RadioAlpha">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Alpha</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RadioDelta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delta</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="RadioBeta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Beta</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="RadioTheta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>96</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Theta</string>
      </property>
     </widget>
    </widget>
    <widget class="QRadioButton" name="BatteryRadio7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BatteryRadio8">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="BatteryRadio5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>140</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BatteryRadio6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BatteryRadio3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>180</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BatteryRadio4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="BatteryRadio1">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="BatteryRadio2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>200</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="StartSavedSession">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>START SAVED SESSION</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>ADMIN CONTROLS</string>
    </property>
    <widget class="QSpinBox" name="Battery">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="rightEarpiece">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>9</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="leftEarpiece">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>45</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9</number>
     </property>
     <property name="value">
      <number>9</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>right earpiece</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>left earpiece</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="Output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;OUTPUT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
